crates/cheatnet/tests/cheatcodes/get_class_hash.rs

```diff
@@ -0,0 +1,46 @@
+use crate::common::{get_contracts, state::create_cheatnet_state};
+use cheatnet::{
+    conversions::{class_hash_to_felt, felt_from_short_string, felt_selector_from_name},
+    rpc::call_contract,
+};
+
+#[test]
+fn get_class_hash_simple() {
+    let mut state = create_cheatnet_state();
+
+    let contracts = get_contracts();
+    let contract_name = felt_from_short_string("HelloStarknet");
+    let class_hash = state.declare(&contract_name, &contracts).unwrap();
```

Comment:
I would extract the `declare` method which will take shortstring as an argument

Reply:
Should be considered in #489